content/format-guide.md

```diff
@@ -26,15 +26,15 @@ The questions:
 
 Running the session:
 
-- There are two staff members required for each session, one facilator and one runner. The facilitator should be the same person who introduced the speaker.
-- When the talk is over, the facilator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
-- Each room will have 2 hand held microphones. The facilator should hold on to one of them, and the runner holds the other.
+- There are two staff members required for each session, one facilitator and one runner. The facilitator should be the same person who introduced the speaker.
+- When the talk is over, the facilitator invites people in the audience who would like to participate in the discussion to come forward and form a circle.
+- Each room will have 2 hand held microphones. The facilitator should hold on to one of them, and the runner holds the other.
 - When someone wants to speak, the runner holds the microphone up to their mouth so that they can. Do not hand the microphone over. Most people don't know how to hold one properly, and you might have to wrestle them to get it back.
```

Comment:
Possibly, change the last three words to ". . . they can speak." e.g. When someone wants to speak, the runner holds the microphone up to their mouth so they can speak.